### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by jmewes**
Fix to typos in the documentation.

### CODE REVIEW

**Review by xu-baolin — APPROVED** [review]
LGTM

File: packages/flutter/lib/src/widgets/router.dart [hunk 1]
```diff
@@ -370,7 +370,7 @@ class Router<T> extends StatefulWidget {
     scope.routerState._setStateWithExplicitReportStatus(_IntentionToReportRouteInformation.must, callback);
   }
```
Comment (oy-paddy) [inline packages/flutter/lib/src/widgets/router.dart:372]: test

**Comment by xu-baolin** [comment]
I am curious that what causes us 'to to' typos, I had seen the same typos before.

**Comment by jmewes** [comment]
> I am curious that what causes us 'to to' typos, I had seen the same typos before.

I would say that "double word typos" are a common typing error. When I do such mistakes, I guess it happens like this:

1. Start writing a sentence
2. Think about the bigger picture
3. Finish writing the sentence

In step 3 I forgot where I stopped typing.

## Automated checks

In this pull request @Hixie considered whether I actually might be a good idea to run tests for this kind of error: https://github.com/flutter/flutter/pull/81325

It could be done with scripts like this:

- https://opendev.org/openstack/searchlight/commit/5252235d9403fd8a57c2134d1cbd93ec40f7ad68
- https://unix.stackexchange.com/questions/256409/command-line-method-to-find-repeat-word-typos-with-line-numbers

Do you already have static code analysis scripts as part of the build process?

### Prototype

I don't know whether it makes sense to introduce such a tool, but the "double word error" seems like being present a few times more.

```
laptop-jm:flutter (stable) janux$ time find . -name "*.dart" | xargs -n 1 pcregrep -Min --color=auto '\b([^[:space:]]+)[[:space:]]+\1\b' | grep "///" > /tmp/potential-double-word-typos.txt

real	0m3,129s
user	0m2,714s
sys	0m0,740s
```

Result (including false positives but also a dozen double word errors):

[potential-double-word-typos.txt](https://github.com/flutter/flutter/files/6465819/potential-double-word-typos.txt)

**Comment by Hixie** [comment]
test-exempt: only affects comments

**Pull request merged**